app/src/component/fileItem/view.js

```diff
@@ -52,18 +54,21 @@ class FileItem extends React.PureComponent {
     } else if (claim === null) {
       description = 'This address contains no content.';
     }
-    
+
     return (
-      <TouchableOpacity style={style} onPress={() => {
-            this.props.navigation.navigate('File', { uri: uri });
-          }
-        }>
-        <FileItemMedia title={title} thumbnail={thumbnail} resizeMode="cover" />
-        <FilePrice uri={uri} style={discoverStyle.filePriceContainer} textStyle={discoverStyle.filePriceText} />
-        <Text style={discoverStyle.fileItemName}>{title}</Text>
-        {channelName &&
-          <Text style={discoverStyle.channelName}>{channelName}</Text>}
-      </TouchableOpacity>
+      <View style={style}>
+        <TouchableOpacity style={discoverStyle.container} onPress={() => {
```

Comment:
You will probably want a custom `Button` component soon to avoid always passing in the style object